src/components/AboutPage/components/AboutList/AboutList.scss

```diff
@@ -0,0 +1,13 @@
+.about-list {
+  margin-bottom: 3rem;
+}
+
+.about-list-grid {
+  &:nth-child(1) {
+    margin-left: auto;
+  }
+  &:nth-child(2) {
+    margin-right: 23.26rem;
```

Comment:
23.26?

Reply:
This is a feature for creating two rows with different number of elements in it.
In this case 1st row - 2 elements
2nd - 4
It's temporary solution will be fix in the future.